entity-framework/core/miscellaneous/configuring-dbcontext.md

```diff
@@ -19,7 +19,7 @@ While any pattern that provides the necessary configuration information to the `
 
 `DbContext` must have an instance of `DbContextOptions` in order to perform any work. The `DbContextOptions` instance carries configuration information such as:
 
-- The database provider to use, typically selected by invoking a method such as `UseSqlServer` or `UseSqlite`
+- The database provider to use, typically selected by invoking a method such as `UseSqlServer` or `UseSqlite`. These extension methods require the corresponding provider package, such as `Microsoft.EntityFrameworkCore.SqlServer` and `Microsoft.EntityFrameworkCore.Sqlite`. A `using` statement for `Microsoft.EntityFrameworkCore` is also required.
```

Comment:
Just nitpicking:
- and —> or
- Using statement or import namespace? Using statement is ambiguous